proto/aserto/directory/reader/v3/reader.proto

```diff
@@ -244,9 +248,9 @@ message GetRelationRequest {
 }
 
 message GetRelationResponse {
-    // array of relation instances
-    repeated aserto.directory.common.v3.Relation results        = 1;
-    // map of materialized relation objects
+    // relation instance
+    aserto.directory.common.v3.Relation result                  = 1;
```

Comment:
@gertd Should this be defined as `optional`? Or do we want the v3 GetRelation to return a not-found error if no matches exists, similar to GetObject?

Reply:
I would return a NOT_FOUND error